Automattic/jetpack
# Various: Fix PHP warnings

Fixes two warnings:

Warning: Illegal offset type in isset or empty in /srv/public/wp-content/plugins/jetpack/modules/sharedaddy/sharing-service.php on line 307

E_WARNING: /home/public_html/wp-content/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-instagram-gallery.php:137 - Invalid argument supplied for foreach()


#### Changes proposed in this Pull Request:

* Validation of variable type before use.

#### Jetpack product discussion

#### Does this pull request change what data or activity we track or use?
n/a

#### Testing instructions:

* For the Sharing one, connected JP, activate Sharing and hit an URL like https://continuing-sacred.jurassic.ninja/2020/07/09/hello-world/?share[0]=twitter&share[1]=facebook&nb=1
(notice the `share[0]` and `share[1]` values.

#### Proposed changelog entry for your changes:
* Bug Fix: Prevent PHP warnings in two instances.
